src/executors/default.yml

```
@@ -11,3 +12,7 @@ parameters:
     description: >
       Pick a specific circleci/python image variant:
       https://hub.docker.com/r/cimg/python/tags
+  resource_class:
+    description: Specify the resource class for Docker Executor
+    type: string
+    default: "small"
```

nit: missing newline here.

I like this.
thought:
you could add validation as an enum here as well; we do something like this in one of our internal orbs, something like

```suggestion
    type: enum
    enum: ["small", "medium", "medium+", "large", "xlarge", "2xlarge", "2xlarge+"]
    default: "small"
```

---

fixed in 1794c13f46cd276f6ba2df5940a1faf3e20aafee 🐮

---

↑ :pray: @wyardley

---

(btw, I'm not a maintainer or Circle employee, so I can't approve this one for you -- lgtm though)

Also, you have a minor typo in the PR title (paramaters vs parameters).